### PR TITLE
feat(playground): enable nursery lint rules

### PR DESCRIPTION
**Pull request by ematipico**
## Summary

This PR adds a new checkbox to the playground to enable nursery rules. This allows us to debug these rules that are still in development. 

I didn't bother much about where to put that checkbox... I am sure we would need an overhaul of the UI, in a separate place.

## Test Plan

Play with the deployed background

### CODE REVIEW

File: website/playground/package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
 		"build:wasm": "wasm-pack build --out-dir ../../npm/wasm-web --target web --release --scope rometools ../../crates/rome_wasm",
 		"build:wasm-dev": "wasm-pack build --out-dir ../../npm/wasm-web --target web --dev --scope rometools ../../crates/rome_wasm",
 		"format": "cargo rome-cli-dev format --write ./src",
-		"check": "cargo rome-cli-dev check ./src",
+		"check": "cd ../../ && cargo rome-cli-dev check ./website/playground/src",
```
Comment (MichaReiser) [inline website/playground/package.json:13]: Why has it become necessary to traverse upwards?

File: website/playground/package.json [hunk 2]
```diff
@@ -36,7 +36,6 @@
 		"@vitejs/plugin-react": "^1.0.7",
 		"autoprefixer": "^10.4.2",
 		"postcss": "^8.4.6",
-		"rome": "next",
```
Comment (MichaReiser) [inline website/playground/package.json:39]: What's the reason behind removing the `rome` dependency

File: website/playground/src/NurseryRules.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
+interface Props {
+	setEnabledNurseryRules: (b: boolean) => void;
+	enabledNurseryRules: boolean;
+}
+
+export default function NurseryRules({
```
Comment (MichaReiser) [inline website/playground/src/NurseryRules.tsx:6]: image

The check box isn't properly aligned with the other check boxes

File: website/playground/src/romeWorker.ts [hunk 4]
```diff
@@ -81,31 +82,46 @@ self.addEventListener("message", async (e) => {
...
+				configuration.linter = {
+					enabled: true,
+					rules: {
+						nursery: {
+							noNewSymbol: "error",
+							noDangerouslySetInnerHtml: "error",
+							noUnusedVariables: "error",
+							noUnreachable: "error",
+							useCamelCase: "error",
 						},
```
Comment (MichaReiser) [inline website/playground/src/romeWorker.ts:119]: Is there no other way to enable all nursery rules? How to we prevent having to change this list with every new nursery rule that we add (or remove)?

**Review by MichaReiser — CHANGES_REQUESTED** [review]

File: website/playground/package.json (hunk 1 above)
Comment (ematipico) [reply website/playground/package.json:13]: This is an existing bug we have in the resolution of the configuration. At the moment we are only able resolve the configuration file from the working directory.

I am already working on fixing the issue, but it requires some refactor on our file system. This is a temporary workaround to let the command to pick up the `rome.json` file

File: website/playground/package.json (hunk 2 above)
Comment (ematipico) [reply website/playground/package.json:39]: Because we don't use it

File: website/playground/src/romeWorker.ts (hunk 4 above)
Comment (ematipico) [reply website/playground/src/romeWorker.ts:119]: Unfortunately, it's not possible at the moment. We can investigate how to solve this issue

File: website/playground/src/NurseryRules.tsx (hunk 3 above)
Comment (ematipico) [reply website/playground/src/NurseryRules.tsx:6]: I am not an expert of tailwind. Do you know how to fix it?

Comment (MichaReiser) [reply website/playground/src/NurseryRules.tsx:6]: I have no prior experience with tailwind. I do think that our playground, being actively promoted on the website, is worth spending time on to ensure it looks decent, similar to diagnostics which we polish too.

**Review by MichaReiser — APPROVED** [review]
Nit: Maybe rename it to *nursery lint rules* to make it clear that it's about linting and not formatting.

**Pull request merged**